pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.3.1
```

Comment:
We should update the multi-snSS2 Overview and Methods docs with the new pipeline version.